### ISSUE TITLE
Skip links

### ISSUE THREAD

**Issue by Eiriken** [woocommerce/storefront#1885]
Hi!
I'm trying to make Storefont accessible to all, and I cannot see that Storefront has a **Skip links** function (https://make.wordpress.org/accessibility/handbook/markup/skip-links/). Can you please add that fuction? This function is required by law in Norway and other countries that are to follow the WCAG accessibility standard.

**Comment by Aljullu** [comment]
Hi @Eiriken, Storefront does have Skip links, you can see them being defined here:

https://github.com/woocommerce/storefront/blob/45bd1c39487518daf40e59a476b5bf8a834b104c/inc/storefront-template-functions.php#L288-L301

I'm going to close this issues because there isn't anything actionable for us. But feel free to reopen if you think that's a mistake. Thank you!

**Comment by Eiriken** [comment]
Ok. Thank you for your reply! It's strange, because the Skip links
function/button does not appear when I press tab at the beginning of my
website (tried Chrome and Edge). Do you know what can cause this? Website:
www.bil-teori.no

**Comment by Aljullu** [comment]
It's strange, they work fine in Storefront, as you can see in the demo site:

https://themes.woocommerce.com/storefront/

Maybe you are using a child theme or by any chance are rewriting the `storefront_skip_links()` function?

I would recommend following the steps of the [WooCommerce Self-Service Guide](https://woocommerce.com/document/woocommerce-self-service-guide/), that will allow you to diagnose where the issue is.

Hope it helps!

**Comment by Eiriken** [comment]
Ok. Thanks for the tips :) I'm not very into coding, but I will try to see
if something happened to the skip_links code. I tried to find the code in
the functions file of both the child and mother theme, but couldn't find
it. Should I look somewhere else?

fre. 28. jan. 2022, 13:31 skrev Albert Juhé Lluveras <
***@***.***>:

**Comment by Aljullu** [comment]
For Storefront, it should be in `storefront/inc/storefront-template-functions.php` (line 297 and 298). If it isn't, somebody must have modified that file, so I would contact the developers that might have done that. If the skip links are in the file I mentioned, then it's probably an issue with the child theme you are using, so you should contact the devs of that theme.

**Comment by Eiriken** [comment]
Thanks a lot! I found the code, so that's good. I have created a child
theme on my own (without devs), but there are not a lot of extra code in
the child functions file (39 lines). So do you think some of the code in
the child functions file could be the reason for it? Or could plugins
destroy the skip links function?

Sorry for the newbie questions....

fre. 28. jan. 2022, 14:58 skrev Albert Juhé Lluveras <
***@***.***>:

**Comment by Aljullu** [comment]
The best way to know is following the [_Plugin and theme conflict_ section](https://woocommerce.com/document/woocommerce-self-service-guide/#section-4) of the WooCommerce Self-Service Guide, this will allow you to know what's causing the issue.

**Comment by Eiriken** [comment]
Ok, thanks!

lør. 29. jan. 2022, 15:11 skrev Albert Juhé Lluveras <
***@***.***>:
